packages/compartment-mapper/src/policy.js

```diff
@@ -0,0 +1,305 @@
+// @ts-check
+
+/** @typedef {import('./types.js').PolicyIdentityComponents} PolicyIdentityComponents */
+
+const { create, entries, assign, keys, freeze } = Object;
+const { stringify } = JSON;
+
+const copyGlobals = (globals, list) => {
+  const g = create(null);
+  if (list && list.length > 0) {
+    for (let k = 0; k < list.length; k += 1) {
+      const key = list[k];
+      // If an endowment is missing, global value is undeinfed.
+      // This is an expected behavior if globals are used for platform feature detection
+      g[key] = globals[key];
```

Comment:
needs to bind. here or by the compartment-mapper caller?

Reply:
Consensus from the Endo call today is that we can defer the responsibility to bind endowments to the user of the compartment mapper.